src/Validation/Validator.php

```diff
@@ -212,21 +212,6 @@ public function setStopOnFailure(bool $stopOnFailure = true)
         return $this;
     }
 
-    /**
-     * Validates and returns an array of failed fields and their error messages.
-     *
-     * @param array $data The data to be checked for errors
-     * @param bool $newRecord whether the data to be validated is new or to be updated.
-     * @return array[] Array of failed fields
-     * @deprecated 3.9.0 Renamed to {@link validate()}.
-     */
-    public function errors(array $data, bool $newRecord = true): array
```

Comment:
Do we want to document each of these removals separately or have a more generalized statement about applications needing to have 0 deprecations before they are ready to upgrade?

Reply:
When I setup the 5.0 docs, that's what 4.0 did. I assumed it would be the same.

Reply:
We already have explicit deprecation warnings for all(most) deprecations, so a general statement should suffice.